packages/internals/src/__tests__/getSchema.test.ts

```diff
@@ -34,20 +35,30 @@ async function testSchemaPath(fixtureName: string, schemaPathFromArgs?: string)
   }
 
   if (typeof asyncResult === 'string') {
-    asyncResult = toUnixPath(path.relative('.', asyncResult))
+    asyncResult = stripAnsi(toUnixPath(path.relative('.', asyncResult)))
   }
 
   if (asyncResult instanceof Error) {
-    asyncResult.message = toUnixPath(asyncResult.message.replace(__dirname, '.'))
+    asyncResult.message = stripAnsi(toUnixPath(asyncResult.message.replace(__dirname, '.')))
   }
 
   return asyncResult
 }
 
-it('returns null if no schema is found', async () => {
+it('throws error if schema is not found', async () => {
   const res = await testSchemaPath('no-schema')
 
-  expect(res).toMatchInlineSnapshot(`null`)
+  expect(res).toMatchInlineSnapshot(`
+    [Error: Could not find a schema.prisma file that is required for this command.
```

Comment:
New error message when schema is not found at default paths

Reply:
Is "schema.prisma file" still correct in this new world?
Maybe just "Prisma Schema" instead?